src/core/README.md

```diff
@@ -1,4 +1,6 @@
-# Overview
+# gRPC core library
```

Comment:
s/core/Core/g

Reply:
From a tech writing point of view, there are a couple of reasons to leave "core" in lowercase. For one, "core" isn't a proper name. IMHO, we're using "core" as a qualifier. But I'll admit that it's a gray area. I'd prefer keeping is in lowercase (which is how it is in the top-level README). Ok?

Reply:
CC advocate of the name "Core" @nicolasnoble .

Reply:
Oh, I think that there might be the confusion between two possible interpretations:

1. Core as in "gRPC Core", or
2. Core as in the "core library" among all the libraries made available from this repo.

The title used in the heading of this document refers to (2). But yes, I'm eager for @nicolasnoble's input. (Thank's for cc  'ing him.)